P13GoTo/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿// See https://aka.ms/new-console-template for more information
+Console.WriteLine("Let's Play a game! Guess the number 1 - 100 or ELSE!");
+
+Random randomNr = new Random();
+int randomNumber = randomNr.Next(1, 100);
+int Guesses = 10;
+int Question = int.Parse(Console.ReadLine());
+bool Win = (Question == randomNumber);
```

Comment:
I recommend you to remove these two lines. You don't need to ask for input before the loop and you never do anything with the `Win` variable.